src/ducks/settings/HarvestBankAccountSettings.jsx

```diff
@@ -78,6 +78,10 @@ const HarvestLoader = ({ connectionId, children }) => {
                   return <HarvestError />
                 }
 
+                if (!data) {
+                  return null
+                }
+
```

Comment:
So if I understand well: 
- We can have an account related to a connector that is not installed anymore. 
- In that case we return null. 

Several remarks: 
- I think Query return an empty array, if there is no result. So data.length === 0 should be a better check 
- What are we displaying if we return null here? What do we want to do in that case? 

Is it possible to add a test to be sure we have the correct behavior?